### DISCUSSION TITLE
Add a skript catagory

### DISCUSSION THREAD

**Discussion by AnxeloK** [modrinth/.github#194] [Feature requests]
In the skript catagory you will be able to find scripts that are made with skript like in spigot.

**Comment by piprett** [comment]
Related: modrinth/knossos#733

**Comment by some-du6e** [comment]
skripts are 90% of the time made by skids and its very niche

**Comment by AnOpenSauceDev** [comment]
As the linked issue points out, Skript is a pretty small community of mainly beginner programmers, and datapacks at this point can 1) do _way_ more than skript nowadays, 2) stay mostly backwards/forwards-compatible, and 3) is maintained by mojang themselves.
